src/Hal/Metric/System/Packages/Composer/Packagist.php

```diff
@@ -37,8 +37,10 @@ public function get($package)
         // get latest version
         $latest = '0.0.0';
         foreach ((array)$json->package->versions as $version => $datas) {
-            $version = preg_replace('([^\.\d])', '', $version);
```

Comment:
The only non numeric character that should be accepted is the leading "v" character that is used by convention.
Some packages have changed from a non-leading "v" to a leading "v" format. This is why we need to remove it before running the `version_compare` function.